### ISSUE TITLE
Clarifying Pretrained Model Call Output

### ISSUE THREAD

**Issue by RylanSchaeffer** [MadryLab/robustness#23]
Can someone please clarify what is returned when a pretrained CIFAR10 model is called? Two objects appear to be returned: the class prediction and an image. Is this correct? If so, what is this image? How does the `make_adv` boolean argument affect that output image?

**Comment by RylanSchaeffer** [comment]
Also, is there a distinction between ´model.forward(img)´ and ´model(img)´?

**Comment by andrewilyas** [comment]
Hi Rylan,

For the first question: The method returns a tuple of the form ``(model output, image)``. The first output is just the model logits (and possibly also the representation layer, if the ``with_latent`` option is passed---see the documentation for more on this). The second output is not very useful unless ``make_adv`` is true, in which case it returns the adversarial examples corresponding to the inputted images.

For the second question, this is more of a general PyTorch question: you should always call the module directly (instead of calling the forward function) as the former calls the latter but also handles some other stuff.

**Comment by andrewilyas** [comment]
Closing this for now, let me know if anything else is needed!
